docs/self-hosting.md

```diff
@@ -13,10 +13,61 @@ The fastest way to get your own version of Maybe running is a "one-click deploy"
 
 ## Docker
 
+0. Install docker (cf. the [docker guide](self-hosting/docker.md) for more
+   info)
+
+### Running the docker compose stack
+
+1. Create a new folder inside your working directory (eg. your home directory
+   or `/srv/docker`) named `maybe`
+
+2. Move into the `maybe` folder and copy the `docker-compose.yml` file from the
+   root of the repository
+
+3. Copy the `.env.example` file from the root of the repository and rename it
+   to `.env`
+
+4. Edit the `.env` file and adjust the `SELF_HOSTING_ENABLED` and
+   `SECRET_KEY_BASE` variables
+
+4. Run `docker-compose up -d` to bring up the stack
+
+5. Navigate to http://localhost:3000 in your browser
```

Comment:
this might be a little _too_ prescriptive, though anyone who know's what they're doing will likely just copy the contents on docker-compose.yml and adapt it

Reply:
i'm a big fan of one-liner scripts (over prescriptive setup instructions) for those less savvy users who will just copy the instructions anyways. makes it less likely they'll mess something up when copying instructions.

> successfully get the project running on a fresh VPS

i wonder what the split between "savvy users who understand docker and compose" and "users running on a cheap vps/rpi" 🤔